### DISCUSSION TITLE
The web socket connection terminates more frequently than claimed.

### DISCUSSION THREAD

**Discussion by simensrostad** [hello-nrfcloud/web#913] [General]
Reported during FAE training. Its often an issue that the web socket connection is dropped when the user is browsing the page.
This happens quite often, more often than every 2 hours. 

![image](https://github.com/user-attachments/assets/dc196f2d-7f0a-4332-b908-47f6cb9706b4)

**Comment by simensrostad** [comment]
It was also very common that the user was browsing the map etc... and the device stopped sending data. And, it turned out that the reason why was because the web socket connection was dropped.

**Comment by coderbyheart** [comment]
Can you provide a bit more information about their system?
Which browser?
What kind of internet connection where they using?
What are the errors in the console?

**Comment by simensrostad** [comment]
I don't recall the system they used. But I've seen it on various setups. Most of them were on the nordic nw I belive.

**Comment by coderbyheart** [comment]
I don't see this issue from home, so I suspect that it is the Nordic firewall. Which office was this in?

**Comment by simensrostad** [comment]
> I don't see this issue from home, so I suspect that it is the Nordic firewall. Which office was this in? 

This was from the leangen office.

**Comment by jtguggedal** [comment]
I also see this on mobile phone, using mobile network for connectivity.

OS: Android 13
Browser: Chrome 130.0.6723.102

Steps to reproduce:

1. Open device page on Hello nRF Cloud
2. Lock the screen for a couple of minutes
3. Unlock screen
4. See warning that websocket has been terminated after 2 hours.
5. Need to reload, map settings etc are reset

**Comment by coderbyheart** [comment]
Yeah, I think we can
- change the wording (the phone will terminate it because it does not run in the background)
- implement automatic reconnect (when we detect that the user interacts with the website) #884
